### PR TITLE
Bug/dl 77 bug/dl 86

### PR DESCRIPTION
**Pull request by liza-panineyeva**
- Added tests to wishlist
- Added javascript to handle like functionality without page reload
- Added popover window with info about the page
- Added magazine option
- Minor changes to style of wishlist

### CODE REVIEW

File: templates/wishlist_add.html [hunk 1]
```diff
@@ -13,9 +13,12 @@
                                <p class="form-title">Create new wish</p>
                                    <form class="wishlist" action="" method="post">
                                        {{ form.hidden_tag() }}
+                                       <label style="color:white;font-style:italic;font-size:12px;">{{form.type.label}}</label>
```
Comment (oskam) [inline templates/wishlist_add.html]: I think it's better to follow _loose coupling_ principle and put this styling it in `css` file.

Comment (liza-panineyeva) [reply templates/wishlist_add.html]: ok, thank you 👍

File: forms/forms.py [hunk 2]
```diff
@@ -102,22 +102,30 @@ class PasswordForm(FlaskForm):
 
 
 class WishlistForm(FlaskForm):
+    type = SelectField('Item Type',
+                       choices=[('book', 'book'), ('magazine', 'magazine')],
```
Comment (tdiak) [inline forms/forms.py]: Second item in tuple is a display name. Reaname it to 'Book' and 'Magazine'

File: static/css/authentication.css [hunk 3]
```diff
@@ -58,6 +58,7 @@ form {
     margin-bottom: 5px;
     color: rgba(255, 255, 255, 1);
     outline: 0;
+
```
Comment (tdiak) [inline static/css/authentication.css]: I think this empty line is unecessary

File: static/js/add_like.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        },
+        success: function(response, data) {
+            console.log(response);
+            var obj = JSON.parse(response);
+            $("#likes_"+wish_id).html(obj.num_of_likes);
+        },
+        error: function(error) {
+            console.log(error);
+        }
+    });
```
Comment (tdiak) [inline static/js/add_like.js]: We should remove console.log before we commit :)
Try to add p with class="error" to in error function

File: templates/base.html [hunk 5]
```diff
@@ -56,6 +56,8 @@
 <script type="text/javascript" src="http://ajax.aspnetcdn.com/ajax/jquery.templates/beta1/jquery.tmpl.js"></script>
 <script type="text/javascript" src="/static/js/search.min.js"></script>
 <script type="text/javascript" src="/static/js/search_wish.min.js"></script>
+<script type="text/javascript" src="/static/js/popover.js"></script>
+<script type="text/javascript" src="/static/js/add_like.js"></script>
```
Comment (tdiak) [inline templates/base.html]: We should use  {{ url_for('static' ... :)

File: templates/wishlist.html [hunk 6]
```diff
@@ -18,10 +21,12 @@ <h1>Search</h1>
...
...DI4LDguODI4bDAsMEM0MjMuNzI0LDQzNy40MjcsNDE5Ljc3Miw0NDEuMzc5LDQxNC44OTcsNDQxLjM3OXoiLz4KCTxwYXRoIHN0eWxlPSJmaWxsOiM5NTlDQjM7IiBkPSJNNDE0Ljg5Nyw0NzYuNjlIOTcuMTAzYy00Ljg3NSwwLTguODI4LTMuOTUzLTguODI4LTguODI4bDAsMGMwLTQuODc1LDMuOTUzLTguODI4LDguODI4LTguODI4ICAgaDMxNy43OTNjNC44NzUsMCw4LjgyOCwzLjk1Myw4LjgyOCw4LjgyOGwwLDBDNDIzLjcyNCw0NzIuNzM3LDQxOS43NzIsNDc2LjY5LDQxNC44OTcsNDc2LjY5eiIvPgoJPHBhdGggc3R5bGU9ImZpbGw6Izk1OUNCMzsiIGQ9Ik00MTQuODk3LDMzNS40NDhIMjU2Yy00Ljg3NSwwLTguODI4LTMuOTUzLTguODI4LTguODI4bDAsMGMwLTQuODc1LDMuOTUzLTguODI4LDguODI4LTguODI4ICAgaDE1OC44OTdjNC44NzUsMCw4LjgyOCwzLjk1Myw4LjgyOCw4LjgyOGwwLDBDNDIzLjcyNCwzMzEuNDk2LDQxOS43NzIsMzM1LjQ0OCw0MTQuODk3LDMzNS40NDh6Ii8+Cgk8cGF0aCBzdHlsZT0iZmlsbDojOTU5Q0IzOyIgZD0iTTQxNC44OTcsMzAwLjEzOEgyNTZjLTQuODc1LDAtOC44MjgtMy45NTMtOC44MjgtOC44MjhsMCwwYzAtNC44NzUsMy45NTMtOC44MjgsOC44MjgtOC44MjggICBoMTU4Ljg5N2M0Ljg3NSwwLDguODI4LDMuOTUzLDguODI4LDguODI4bDAsMEM0MjMuNzI0LDI5Ni4xODUsNDE5Ljc3MiwzMDAuMTM4LDQxNC44OTcsMzAwLjEzOHoiLz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8Zz4KPC9nPgo8L3N2Zz4K" style="width:25px"/>{% endif %}
```
Comment (tdiak) [inline templates/wishlist.html]: Maybe we should change base64  to jpg/png file. What do You think about it ?

File: views/index.py [hunk 7]
```diff
@@ -44,6 +45,8 @@
 
 @library.route('/')
 def index():
+    # db.drop_all()
+    # db.session.commit()
```
Comment (tdiak) [inline views/index.py]: Why You append this comments ?

**Pull request merged**